client/src/components/ReviewVotes.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react"
+
+const ReviewVotes = props => {
+
+  const [voteStatus, setVoteStatus] = useState(props.review.voteStatus)
+  const [voteCount, setVoteCount] = useState(props.review.votes)
+  const [showSignInMessage, setShowSignInMessage] = useState(false)
```

Comment:
while this does work, it unfortunately means we are tracking the voteInformation in two places. Up in the Showpage under the reviews array, and down here. After a vote, what is in the show page and here conflict, which we need to avoid to circumvent potential future bugs.

As tedious as it sounds, you need to update/maintain  information in the Sandwhich show state. You may decide to store `reviews` as a separate chunk of state for convenience if you like, or stick with what you have and remove the state here